loadgen/workflow_executor.go

```diff
@@ -0,0 +1,41 @@
+package loadgen
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.temporal.io/sdk/client"
+)
+
+// WorkflowExecutor is a very dumb executor that does nothing but run the specified workflow.
+// Scenarios using it are expected to implement all their driver logic as a workflow.
+type WorkflowExecutor struct {
+	WorkflowType       string
+	WorkflowArgCreator func(info ScenarioInfo) []interface{}
+}
+
+func (w WorkflowExecutor) Run(ctx context.Context, info ScenarioInfo) error {
+	// TODO: ??
+	//executorTimeout := info.ScenarioOptionDuration(scenarios.ExecutorTimeoutFlag, 1*time.Hour)
```

Comment:
Need to decide how to fix this. It's not really a scenario option, but an executor-specific option.

Reply:
I think we have duration on the main args right, can't you reuse that?

Reply:
I could, it's a little unspecific though. Since, it's a timeout now, really not a duration.